.bldr.toml

```diff
@@ -23,9 +23,7 @@ paths = [
   "api/config/platform/*",
   "api/config/shared/*",
   "api/interservice/pg_sidecar/*",
-  "components/automate-gateway/api/authz/*",
```

Comment:
Just like we have to regenerate from proto file changes, when there are actual endpoint changes this may impact the bldr config, so `generate_bldr_config` regenerates this file.